pom.xml

```diff
@@ -116,7 +116,8 @@
 
     <processorVersion>1.3</processorVersion>
 
-    <doxiaVersion>1.6</doxiaVersion>
+    <doxiaVersion>1.8</doxiaVersion>
+    <doxiaRendererVersion>1.8.1</doxiaRendererVersion>
```

Comment:
If we really need both `doxiaVersion` and `doxiaRendererVersion` please add a comment explaining why.

Reply:
I think they fixed important stability issues, especially the last two:
https://issues.apache.org/jira/browse/DOXIASITETOOLS-187
https://issues.apache.org/jira/browse/DOXIASITETOOLS-179
https://issues.apache.org/jira/browse/DOXIASITETOOLS-189

Reply:
So `org.apache.maven.doxia:doxia-site-renderer` has independent release cycle from other `org.apache.maven.doxia:*` artifacts that we use? If so, this is what should be noted on line 120 of our pom.xml.